javascript-data-model/order-history.js

```diff
@@ -0,0 +1,75 @@
+var orderHistory = [
+  {
+    orderPlaced: 'August 4, 2020',
+    totalPrice: 34.00,
+    totalItems: 1,
+    sentTo: 'JS Masher',
+    orderNumber: '144-3941689-8772232',
+    delivered: 'Aug 8, 2020',
+    directHand: false,
+    items: [{
+      name: 'JavaScript for impatient programmers',
+      by: 'Rauschmayer, Dr Axel',
+      returnWindowClose: 'Return window closed on Sep 7, 2020',
+      price: 34.00
+    }]
+  },
+  {
+    orderPlaced: 'July 19, 2020',
+    totalPrice: 44.53,
+    totalItems: 1,
+    sentTo: 'JS Masher',
+    orderNumber: '113-9984268-1280257',
+    delivered: 'Jul 20,2020',
+    directHand: true,
+    items: [{
+      name: 'The Timeless Way of Building',
+      by: 'Alexander, Christopher',
+      returnWindowClose: 'Return window closed on Aug 19, 2020',
+      price: 41.33
+    }]
+  },
+  {
+    orderPlaced: 'July 4, 2020',
+    totalPrice: 17.22,
+    totalItems: 1,
```

Comment:
This property can be removed, since the same information can be retrieved from the length of the `items` array 🤓